Applications/Terminal/Terminal.cpp

```diff
@@ -388,8 +430,11 @@ void Terminal::escape$M(const ParamVector& params)
     count = min(count, max_count);
 
     dbgprintf("Delete %d line(s) starting from %d\n", count, m_cursor_row);
-    // FIXME: Implement.
-    ASSERT_NOT_REACHED();
+    for (word i = 0; i < count; ++i)
+        delete m_lines[m_cursor_row + i];
```

Comment:
Here we are instead deleting lines but never creating any new ones.
The `m_lines` array must always have `m_rows` valid entries.